packages/adapter-static/README.md

```diff
@@ -38,6 +38,25 @@ The directory to write static assets (the contents of `static`, plus client-side
 
 Specify a fallback page for SPA mode, e.g. `index.html` or `200.html` or `404.html`.
 
+### entry_file_names
+
+A optional function that is called per entry to return the destination to write to.
+
+Default value:
+
+```javascript
+/** @type {(path: string, is_html: boolean) => string} */
+function defaultEntryFileNames(path, is_html) {
```

Comment:
We generally pass just a single argument to public APIs for backwards compatibility that way if we want to add a new argument or remove an argument it's easier to do with less impact on users. You can then destructure it for almost the same usage:
```
function defaultEntryFileNames({path, is_html}) {
```